core/src/com/unciv/ui/utils/ExtensionFunctions.kt

```diff
@@ -177,13 +206,37 @@ fun String.toLabel(fontColor: Color = Color.WHITE, fontSize:Int=18): Label {
     return Label(this.tr(), labelStyle).apply { setFontScale(fontSize/Fonts.ORIGINAL_FONT_SIZE) }
 }
 
-fun Label.setFontColor(color: Color): Label { style= Label.LabelStyle(style).apply { fontColor=color }; return this }
+/**
+ * Translate a [String] and make a [CheckBox] widget from it.
+ * @receiver The text to use as [CheckBox] label
+ * @param initialState The initial checked state
+ * @param changeAction A callback to call on change with a boolean lambda parameter containing the current [isChecked][CheckBox.isChecked]
+ * @return the initialized [CheckBox]
```

Comment:
Again, TMI. String.toCheckBox returns THE CHECKBOX? initialState is "The initial checked state"? Golly goshness, Batman!
The changeAction is the only non-trivial information here.

Reply:
True.